addons/mail/wizard/mail_template_preview_views.xml

```diff
@@ -6,39 +6,36 @@
             <field name="model">mail.template.preview</field>
             <field name="arch" type="xml">
                 <form string="Email Preview">
-                    <h3>Preview of <field name="mail_template_id" readonly="1" nolabel="1" options="{'no_open' : True}"/></h3>
-                    <div class="alert alert-danger" role="alert" attrs="{'invisible' : [('error_msg', '=', False)]}">
-                        <field name="error_msg"/>
-                    </div>
-                    <field name="no_record" invisible="1"/>
-                    <div class="container">
-                        <div class="row">
-                            <span class="col-md-5 col-lg-4 col-sm-12 ps-0">Choose an example <field name="model_id" readonly="1"/> record:</span>
-                            <div class="col-md-7 col-lg-6 col-sm-12 ps-0">
-                                <field name="resource_ref" readonly="False" class="w-100"
-                                    options="{'hide_model': True, 'no_create': True, 'no_open': True}"
-                                    attrs="{'invisible': [('no_record', '=', True)]}"/>
-                                <b attrs="{'invisible': [('no_record', '=', False)]}" class="text-warning">No record for this model</b>
-                            </div>
+                    <sheet>
+                        <div class="oe_title">
+                            <h3>Preview of <field name="mail_template_id" readonly="1" nolabel="1" options="{'no_open' : True}"/></h3>
                         </div>
-                        <div class="row">
-                            <span class="col-md-5 col-lg-4 col-sm-12 ps-0">Force a language: </span>
-                            <div class="col-md-7 col-lg-6 col-sm-12 ps-0">
-                                <field name="lang" placeholder="Select a language" class="w-100"/>
-                            </div>
+                        <div class="alert alert-danger" role="alert" attrs="{'invisible' : [('error_msg', '=', False)]}">
```

Comment:
While we are at it, could you please rework the attrs so they are formatted the same way?

I see some:
`attrs="{'invisible' : [('error_msg', '=', False)]}"` with double spaces around the `:`
and
`attrs="{'invisible':[('email_from','=', False)]}"` with no spacing.

The correct way is one space after the `:` (and none before).
So this one is correct:
`attrs="{'invisible': [('no_record', '=', True)]}"`

Not a big deal, but it looks slightly better :)

Reply:
Thanks, yes I should have reformatted them. I have reformatted all attributes of the form containing python code.